client/src/pages/Feed.js

```diff
@@ -299,7 +299,9 @@ const Feed = (props) => {
     dispatchAction(SET_VALUE, "applyFilters", true);
   };
 
-  const handlePostLike = async (postId, liked) => {
+  const handlePostLike = async (postId, liked, create) => {
+    sessionStorage.removeItem("likePost");
```

Comment:
not sure why this is being added into and removed from the session storage?

Reply:
if it is a guest user, feed redirects to the login page and it saves the liked post in storage, after returns, it tries to do the like If the user logged in.

Reply:
ok yeah I figured that, does it also work for the social media logins? otherwise might need another way to solve this

Reply:
This looks at the isAuthenticated variable to see if the user logged in, it should work with any login.

Reply:
well the social meida logins redirect to the social media URL via OAuth and come back to our app so I suppose that is not the same session hence it won't work with session storage? I remember working on something similar and using query params on the callback URL for that.. but I'm not 100% sure that's why I''m asking, so you have tested this?

Reply:
Hmm doesn't seem to be consistently working and ideally we'd also redirect the user to the post page in such a case. However, the most important thing for soft launch is that the liking/unliking is working when logged in on the post page. We can iterate on this part.